docs/running-locally.md

```diff
@@ -108,6 +108,8 @@ You can use the `TARGET_PLATFORM` environment variable to compile images for spe
 make argoexec-image TARGET_PLATFORM=linux/arm64,linux/amd64
 ```
 
+!!! Note "expected 'package', found signal..." If you see this error, you need to set git `core.symlinks=true`, just run `git config --global core.symlinks true` or `git -c core.symlinks=true clone`.
```

Comment:
```suggestion
!!! Note "Error `expected 'package', found signal_darwin`"
    You may see this error if symlinks are not configured for your `git` installation.
    Run `git config core.symlinks true` to correct this.
```

- Admonitions have 4 space indented content
- Use backticks for error messages and `git`
- Only suggest the per-repo configuration. Global is not necessary and this error message will happen _after_ a clone
- Remove the word "just", per [k8s style guide](https://kubernetes.io/docs/contribute/style/style-guide/#avoid-words-that-assume-a-specific-level-of-understanding)